app/javascript/components/model/PaginationTableModel.js

```diff
@@ -2,17 +2,20 @@ import { computed, observable, decorate} from "mobx"
 import Result from "./Result"
 
 export default class PagingTableModel {
-  section = null // String
-  filter = null // String
-  daac = null // String
-  campaign = null // String
-  colorCode = null // String
-  sortColumn = null // String
-  sortOrder = "asc" // String
-  currentPage = 1 // Int
-  result = null // Result
-  loading = false // Bool
-  pageSize = 10 // Int
+
+  constructor() {
+    this.section = null // String
```

Comment:
Did the other way fail?

Reply:
Not sure, when I checkout out your branch and ran, I had no failures but ~40 errors IIRC. I had in  my pocket a branch that one of my devs had been working on the react stuff, and when I merged yours and theirs all was well.